rt/src/runtime/time.rs

```diff
@@ -1,19 +1,17 @@
 use crate::mem::{Float, Int};
 use crate::state::State;
+use rustix::time;
 use std::mem::MaybeUninit;
 
 fn utc() -> f64 {
-    unsafe {
-        let mut ts = MaybeUninit::uninit();
-
-        if libc::clock_gettime(libc::CLOCK_REALTIME, ts.as_mut_ptr()) != 0 {
-            panic!("clock_gettime() failed");
-        }
-
-        let ts = ts.assume_init();
-
-        ts.tv_sec as f64 + (ts.tv_nsec as f64 / 1_000_000_000.0)
+    let result = std::panic::catch_unwind(|| {
```

Comment:
ohhh, so does it mean I won't need to keep this panic! also? `panic!("clock_gettime() failed");`

Reply:
@thaodt The `panic` can also be removed, because by the looks of it rustix already panics if needed:

- https://github.com/bytecodealliance/rustix/blob/40653851826679938449410b94d25e7aa7596561/src/backend/linux_raw/vdso_wrappers.rs#L48
- https://github.com/bytecodealliance/rustix/blob/40653851826679938449410b94d25e7aa7596561/src/backend/libc/time/syscalls.rs#L120